src/api/posts/src/data/post.js

```diff
@@ -35,6 +35,28 @@ function ensureFeed(feed) {
   return feed instanceof Feed ? Promise.resolve(feed) : Feed.byId(feed);
 }
 
+/**
+ * Determine the type of the post depending on the given URL.
+ * The possible types are 'video' or 'blogpost'.
+ *
+ * @param {*} url
```

Comment:
The `url` is a `string`, right?  Let's use that vs. `*`

Reply:
I updated this change! Thank you for pointing it out 😃